eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -440,17 +458,30 @@ jobs:
 
   # Only upload test results if enabled
   - ${{ if eq(parameters.runTests, 'True') }}:
+    - ${{ if eq(parameters.buildSourceOnly, 'True') }}:
+      - task: PublishTestResults@2
+        displayName: Publish Source Build Smoke Test Results
+        condition: succeededOrFailed()
+        continueOnError: true
+        inputs:
+          testRunner: vSTest
+          testResultsFiles: 'test/**/*.trx'
+          searchFolder: $(sourcesPath)
+          mergeTestResults: true
+          publishRunAttachments: true
+          testRunTitle: SourceBuild_SmokeTests_$(Agent.JobName)
```

Comment:
Could we instead tell the smoke tests project to put the test result files under the known artifacts/TestResults/ location?

Reply:
This is now implemented by https://github.com/dotnet/installer/pull/19290